packages/react-ui/src/theme/sections/typography.js

```diff
@@ -9,117 +11,219 @@ export const themeTypography = {
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(18),
+    lineHeight: 1.333,
+    letterSpacing: '0.15px'
   },
   subtitle1: {
-    fontFamily: 'Montserrat, sans-serif',
-    fontWeight: 600,
-    fontSize: '1rem',
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(16),
     lineHeight: 1.5,
-    letterSpacing: '0.009em',
-    fontSmoothing: 'antialiased'
+    letterSpacing: '0.15px'
   },
   subtitle2: {
-    fontFamily: '"Open Sans", sans-serif',
+    fontFamily: 'Inter, sans-serif',
     fontWeight: 600,
-    fontSize: '0.875rem',
-    lineHeight: 1.715,
-    letterSpacing: '0.007em',
-    fontSmoothing: 'antialiased'
+    fontSize: getPixelToRem(13),
+    lineHeight: 1.538,
+    letterSpacing: '0.1px'
   },
   body1: {
-    fontFamily: '"Open Sans", sans-serif',
+    fontFamily: 'Inter, sans-serif',
     fontWeight: 400,
-    fontSize: '1rem',
+    fontSize: getPixelToRem(16),
     lineHeight: 1.5,
-    letterSpacing: '0.028em',
-    fontSmoothing: 'antialiased'
+    letterSpacing: '0.4px'
   },
   body2: {
-    fontFamily: '"Open Sans", sans-serif',
+    fontFamily: 'Inter, sans-serif',
     fontWeight: 400,
-    fontSize: '0.875rem',
-    lineHeight: 1.429,
-    letterSpacing: '0.018em',
-    fontSmoothing: 'antialiased'
+    fontSize: getPixelToRem(13),
+    lineHeight: 1.538,
+    letterSpacing: '0.25px'
   },
   button: {
-    fontFamily: 'Montserrat, sans-serif',
-    fontWeight: 600,
-    fontSize: '0.875rem',
-    lineHeight: 1.715,
-    letterSpacing: '0.018em',
-    textTransform: 'inherit',
-    fontSmoothing: 'antialiased'
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(13),
+    lineHeight: 1.538,
+    letterSpacing: '0.25px',
+    textTransform: 'inherit'
   },
   caption: {
-    fontFamily: '"Open Sans", sans-serif',
-    fontWeight: 600,
-    fontSize: '0.75rem',
-    lineHeight: 1.334,
-    letterSpacing: '0.017em',
-    fontSmoothing: 'antialiased'
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 400,
+    fontSize: getPixelToRem(11),
+    lineHeight: 1.454,
+    letterSpacing: '0.2px'
   },
   overline: {
-    fontFamily: '"Open Sans", sans-serif',
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(10),
+    lineHeight: 1.2,
+    letterSpacing: '2px',
+    textTransform: 'uppercase'
+  }
+};
+
+const customTypography = {
+  body1Italic: {
+    ...baseTypography.body1,
+    fontStyle: 'italic'
+  },
+  body1Strong: {
+    ...baseTypography.body1,
+    fontWeight: 600
+  },
+  body1StrongItalic: {
+    ...baseTypography.body1,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  body2Italic: {
+    ...baseTypography.body2,
+    fontStyle: 'italic'
+  },
+  body2Strong: {
+    ...baseTypography.body2,
+    fontWeight: 600
+  },
+  body2StrongItalic: {
+    ...baseTypography.body2,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  buttonLarge: {
```

Comment:
Are these components-related variables gonna be here in the long-term? Is this the best place for those?

Reply:
They are going to use them across the platform, so I think they should be here. But you are right that button is not the best naming for a reusable variant. I'll talk to them.

Reply:
Button styles are gonna be out